src/main/java/org/jmhsrobotics/frc2024/subsystems/LED/commands/LEDCommand.java

```diff
@@ -0,0 +1,40 @@
+package org.jmhsrobotics.frc2024.subsystems.LED.commands;
+
+import org.jmhsrobotics.frc2024.subsystems.LED.LEDSubsystem;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class LEDCommand extends Command {
+
+	private LEDSubsystem ledSubsystem;
+
+	public LEDCommand(LEDSubsystem ledSubsystem) {
+		this.ledSubsystem = ledSubsystem;
+
+		addRequirements(this.ledSubsystem);
+	}
+
+	@Override
+	public void initialize() {
+		this.ledSubsystem.startLED();
+
+	}
+
+	@Override
+	public void execute() {
+
+		// create a rainbow effect
+		// this.ledSubsystem.rainBow();
+		this.ledSubsystem.setRGB(255, 0, 0);
```

Comment:
If setting a solid color method only needs to be set once within ```initialize``` If we are doing rainbow this is fine to do within excecated as the databuffer for the leds changes every loop.